package-system/pyside2/build_linux.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+# 
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+
+# Get the python executable from the package dependency
+LOCAL_PYTHON37_BIN=$TEMP_FOLDER/python-3.10.5-rev1-linux/python/bin/python3
+
+SCRIPT_PATH=`dirname $0`
+
+
+if [ ! -f "$LOCAL_PYTHON37_BIN" ]
+then
+    echo "Missing 3P dependency of python3.7 $LOCAL_PYTHON37_BIN"
```

Comment:
This should be updated to mention that the dependent is python 3.10

Reply:
Done